docs/how-to/how-to-regenerate-certificates-with-octopus-server-and-tentacle.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to regenerate certificates with Octopus Server and Tentacle
+description: How to regenerate the certificates used for communication between Octopus Server and its Tentacle.
+position: 19
+version: "[3.14,)"
+---
+
+Octopus uses self-signed certificates to securely communicate between Tentacles and the server. Prior to Octopus 3.14, the certificates were SHA1, and following the [shattered](https://octopus.com/blog/shattered) exploit, the certificate generation was upgraded to SHA256. This guide walks through the process of regenerating certificates to use the new algorithm. This is not only useful for upgrading from SHA1 to SHA256, but also if you want to rotate certificates.
+For more information on self-signed certificates, see the [blog post](https://octopusdeploy.com/blog/why-self-signed-certificates) on the topic.
+
+You can view the algorithm used by the Server certificate on the {{Configuration,Certificates}} page. If the algorithm contains `sha1`, we recommend regenerating your certificate.
+
+:::warning
+**Updating an existing Octopus server or Tentacle**
+It's important to consider the impact of updating an existing Octopus server or Tentacle as changes are required to ensure each component trusts the other. Read the information below carefully.
+:::
+
+## Configuring Octopus Server to use a new certificate {#ConfiguringOctopusServerToUseANewCertificate}
+
+This assumes you have already installed Octopus on the target server.
+
```

Comment:
I think a very brief summary of the steps would be nice here. Just a very terse set of dot points so people have a picture of the overall process in their heads.